programs/bpf/tests/programs.rs

```diff
@@ -305,16 +305,29 @@ fn run_program(name: &str) -> u64 {
                     tracer = Some(vm.get_tracer().clone());
                 }
             }
-            deserialize_parameters(
+            assert!(match deserialize_parameters(
```

Comment:
Removing it would mean to remove the `deserialize_parameters` all together in this test, which is possible. The match is necessary as some errors (those coming from `enable_early_verification_of_account_modifications`) are expected here.

Reply:
Yeah I think it seems ok to remove. @jackcmay do you think we could miss important issues in bpf program tests if remove this?